### ISSUE TITLE
Include settings to allow damage while unconscious

### ISSUE THREAD

**Issue by KidIrie** [acemod/ACE3#6929]
**Arma 3 Version:** `1.82.144647` 
**CBA Version:** `3.6.1` 
**ACE3 Version:** `3.12.2`

**Mods:**
```
- CBA_A3
- ace
```

**Description:**
Originally (somewhat) brought up in ticket #6725. If Ace unconscious is enabled, downed units take no damage. The unconscious unit can only die after their revive time runs out. Please make a setting where the user can enable taking damage while unconscious.

I understand the reasoning behind not taking damage while unconscious in a multiplayer environment with people playing co-op against enemy AI. But as a single player, I want to enable unconscious for my AI teammates. Unfortunately this means the enemy will also fall unconscious. I can set Delay Cease Fire of AI for Unconsciousness to 0, so my AI teammates will initially ignore the downed enemy. However when I walk by the downed enemy, the friendly AI register the downed enemy as a "threat" and open fire/throw grenades, injuring and killing me, themselves, and everyone else in the area.

I thought I could live with this but I play a lot of Dynamic Recon Ops missions, where walking up to and searching downed enemies is an important part of every mission. I've been killed by and had many AI teammates killed by fellow AI in this manner more times than I can count now.

I love the Ace medical system and I want to continue using it. As it stands now though, I dread when an enemy goes unconscious on me while I'm near him. It breaks reality that he becomes invincible, I have to run away from him, and then I have to wait around at a distance until his revive timer runs out. I should be able to finish off the enemy. 

Thanks so much for your consideration in this.

**Steps to reproduce:**
Set Max Revive Time in setting to 300 seconds. Place friendly and enemy units in Editor and have them shoot each other up. Advance and hover around an unconscious enemy unit with your squad and watch them blast away at the downed unit (and at you and themselves accidentally) almost non-stop for the full 300 seconds. 

**Where did the issue occur?**
Everywhere

**Comment by bux** [comment]
> not taking damage while unconscious

Don't use Revive for such scenarios.

**Comment by KidIrie** [comment]
You're right, it's a feature request. I posted in the wrong area. Guess it should be moved (?)

In the above 2nd paragraph I did already state that, as a single player, I want to be able to revive my own AI teammates. Unfortunately this means enemy AI must also be set to revive. This is the main issue and why I'm asking for just all AI to be able to revive but also have the user option of taking damage in the meantime. It would seem the simplest solution (though I'm not a programmer) and is realistic. 

I also explained at the end of the 2nd paragraph that friendly AI blasting away at the downed unit is not a bug. As it's been explained to me anyway in ticket #6725, the friendly AI sees the downed enemy as a threat, doesn't know it can't be killed, so continues to non stop fire upon it. You'll see what I mean if you follow the steps to reproduce.
